### PR TITLE
fpga_crihook: fpga_crihook: improve unit tests

### PR DESCRIPTION
**Pull request by bart0sh**
- increased test coverage to 91.4%
- cleaned up the code
- removed unused test data

Fixes: issue #257

### CODE REVIEW

File: cmd/fpga_crihook/main.go [hunk 1]
```diff
@@ -228,7 +227,7 @@ func getStdin(reader io.Reader) (*Stdin, error) {
 	return &stdinJ, nil
 }
 
-func (he *hookEnv) process(reader io.Reader) error {
+func (he *hookEnv) process(reader io.Reader, newPort newPortFun) error {
```
Comment (rojkov) [inline cmd/fpga_crihook/main.go]: Probably this dependency injection could be made only once in `newHookEnv()` to unify it with injecting `utilsexec.Interface`?

Comment (bart0sh) [reply cmd/fpga_crihook/main.go]: thank you for the review! done.

File: cmd/fpga_crihook/main.go [hunk 2]
```diff
@@ -176,7 +174,7 @@ func (he *hookEnv) getFPGAParams(config *Config) ([]fpgaParams, error) {
 			if dev.processed {
 				continue
 			}
-			port, err := fpga.NewPort(deviceName)
+			port, err := newPort(deviceName)
```
Comment (rojkov) [inline cmd/fpga_crihook/main.go]: You can also call `he.newPort(deviceName)` here. No need to change the signature of `getFPGAParams()`.

Comment (bart0sh) [reply cmd/fpga_crihook/main.go]: thanks. done

**Pull request merged**